src/index.css

```diff
@@ -25,27 +25,25 @@ body {
 }
 
 /*========GRID AREA==========*/
-.header-grid {
+/* .header-grid {
   grid-area: header;
-}
+} */
 
-.app-contents-grid {
+/* .app-contents-grid {
   height: 100vh;
   grid-area: appContents;
-}
+} */
 .sidebar-grid {
-  padding-top: 25px;
-  padding-left: 10px;
   width: 245px;
   grid-area: sidebar;
 }
 
-.grid-container {
+/* .grid-container {
   height: 100%;
   display: grid;
   grid-template-columns: 200px;
   grid-template-rows: auto;
   grid-template-areas:
     "sidebar header header header"
     "sidebar appContents appContents appContents";
```

Comment:
in general, it's good practice to not leave commented code in so i think we should delete these unused classes